vividus-plugin-web-app/src/main/java/org/vividus/ui/web/util/SearchAttributesConversionUtils.java

```diff
@@ -98,12 +99,20 @@ private static SearchAttributes convertToSearchAttributes(String searchType, Str
             case "xpath":
                 return new SearchAttributes(ActionAttributeType.XPATH, LocatorUtil.getXPath(searchValue));
             default:
-                return findActionAttributeType(ActionAttributeType.getSearchTypes(), searchType)
-                        .map(type -> new SearchAttributes(type, searchValue))
-                        .orElseThrow(() -> new IllegalArgumentException("Unsupported locator type: " + searchType));
+                return createSearchAttributes(ActionAttributeType.getSearchTypes(),
+                    type -> new SearchAttributes(type, searchValue), "locator", searchType);
         }
     }
 
+    private static SearchAttributes createSearchAttributes(Set<ActionAttributeType> attributeTypes,
+            Function<ActionAttributeType, SearchAttributes> mapper, String operatorType, String searchType)
+    {
+        return findActionAttributeType(attributeTypes, searchType)
+                .map(mapper)
+                .orElseThrow(() -> new IllegalArgumentException(
```

Comment:
It's a breaking change, it should be delivered carefully

Reply:
I would say it will open eyes to those people who hoped that their locators work properly :)